hw3/task1/index.html

```diff
@@ -0,0 +1,305 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Clonify</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div class="container">
+        <aside class="sidebar">
+            <a href="" class="sidebar__logo" src="svg/ui/logo.svg" alt="Logo"></a>
+            <nav class="sidebar__menu">
+                <div class="sidebar__dashboard">
+                    <h2>Dashboard</h2>
+                    <ul>
+                        <li>
+                            <div class="sidebar__li"><img src="svg/ui/dashboard.svg" alt="">
+                                <a class="sidebar__item--chosen" href="">Dashboard</a>
```

Comment:
I expect dashboard.svg and arrow_up.svg inside `<a>`, otherwise they are not clickable